auto-gen.py

```diff
@@ -243,23 +249,24 @@ def write_to_csv(yes_strings_mm_baby, no_write, version):
     print('Building full item list...')
 
     full_list = build_item_list(yes_strings_mm_baby)
-    
+
     print('Full item list built!')
 
     if version:
-        final_list = full_list[full_list['Version Added']==version].copy()
+        final_list = full_list[full_list['Version Added'] == version].copy()
         final_list = final_list[columns]
-        
+
         if no_write:
-            final_list.to_csv(f'./output/NewItems_v{version}.csv', index = False)
+            final_list.to_csv(f'./output/NewItems_v{version}.csv', index=False)
             print(f'Item list written to ./output/NewItems_v{version}.csv')
     else:
         final_list = full_list[columns]
 
         if no_write:
```

Comment:
I think this is actually backwards, don't you want to write if "no_write" **isn't** set?

Reply:
no_write defaults to true (so the default state of the script is to write a file out) and the --no-write flag sets it to False so, not backwards, but maybe a case of bad variable naming :sweat_smile:

Reply:
I see, would've also be fixed by having it default to False, which makes more sense to me? Usually when you specify a flag, that makes it true, it's strange for a flag to be true when not specified.